### ISSUE TITLE
Sample Training Times for CIFAR/ MNIST Models

### ISSUE THREAD

**Issue by stripathi669** [goodfeli/adversarial#2]
Hi ,
I wondering if readers could be made aware of approx training time for a few of the models. Since it will mostly depend on hardware and software environment, maybe the hardware and software systems used in the paper "Generative Adversarial Nets" can be posted.

Thanks

**Comment by anguyen8** [comment]
@stripathi669 : Agreed. Did you find out how long it takes to train the MNIST model? (until you see D(X) = 1/2) Thanks.

**Comment by yaxingwang** [comment]
Hi @anguyen8 , Do you know how to look the value of D(x) in this programs
